rudder-agent/SOURCES/debug-script/tests/rudder_unsent_inventories_agent

```diff
@@ -21,7 +21,7 @@ rudder_unsent_inventories_agent() {
     UNSENT_INVENTORIES_DIR="${RUDDER_VAR}/inventories"
   fi
 
-  COMMAND_OUTPUT=`ls -l "${UNSENT_INVENTORIES_DIR}" 2>&1 | wc -l`
+  COMMAND_OUTPUT=`tail -n 10 "${UNSENT_INVENTORIES_DIR}" 2>&1 | wc -l`
```

Comment:
After looking at the bug, I see that is says "recent inventories"
This means that only inventories from a few days ago should be sent.

Moreover you cannot use tail on a directory

Reply:
Yes that's it. but the script checks if directory result is higher than 1 to show that there is an failed inventory. How can i determine these few days !!!

Reply:
changed.